docs/guide/clustering.md

```diff
@@ -15,13 +15,47 @@ is up to you and your performance needs.
 
 The minimal requirements are:
 
-- Shared HTTP cache (using Varnish)
-- Shared persistence cache and sessions (using Memcached or Redis)
+- [Shared HTTP cache (using Varnish)](http_cache.md#using-varnish)
+- [Shared persistence cache](#shared-persistence-cache) and [sessions](#shared-sessions) (using Memcached or Redis)
 - Shared database (using MySQL/MariaDB)
-- Shared filesystem (using NFS, or S3)
+- [Shared binary files](#shared-binary-files) (using NFS, or S3)
 
 For further details on requirements, see [Requirements page](../getting_started/requirements.md).
 
+It is also recommended to use:
+
+- [Solr](solr.md) for better search and performance
+- a CDN for improved performance and faster ping time worldwide
+- active/passive database for failover
+- more recent versions of PHP and MySQL/MariaDB within [what is supported](../getting_started/requirements.md) for your eZ Platform version to get more performance out of each server. Numbers might vary so make sure to test this when upgrading.
+
+### Shared persistence cache
+
+[Memcached, a distributed caching solution](http://memcached.org/) is the main supported cache solution for clustering.
+
+See [Memcached Cache Adapter in Symfony documentation](https://symfony.com/doc/3.4/components/cache/adapters/memcached_adapter.html#configure-the-connection)
+for information on how to configure Memcached.
+
+!!! note
+
+    To use Memcached, you need to set `cache_service_name` to `cache.memcached`.
```

Comment:
For me it's not an additional information but a step that must be taken so I would move it into the core text.